packages/js-client-sdk/src/lib/CheckoutWithCard.ts

```diff
@@ -195,7 +201,17 @@ export function createCheckoutWithCardElement({
   onBeforeModalOpen,
   onPriceUpdate,
   useAltDomain = true,
+  configs,
 }: CheckoutWithCardElementArgs) {
+  let clientSecret = sdkClientSecret;
```

Comment:
This now just checks if an sdk client secret was passed in and if not generate a base64 encoded string from the configs. If neither are passed in, throw an error.